pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -84,6 +86,15 @@ var (
 )
 
 const (
+	DefaultMachineMutatingHookPath      = "/mutate-machine-openshift-io-v1beta1-machine"
+	DefaultMachineValidatingHookPath    = "/validate-machine-openshift-io-v1beta1-machine"
+	DefaultMachineSetMutatingHookPath   = "/mutate-machine-openshift-io-v1beta1-machineset"
+	DefaultMachineSetValidatingHookPath = "/validate-machine-openshift-io-v1beta1-machineset"
+
+	defaultWebhookName              = "machine-api-operator-webhook"
```

Comment:
nit: this is confusing to me, this should `defaultWebhookServiceName` consistently with `defaultWebhookServiceNamespace`, and so it's clear its difference from `defaultWebhookConfigurationName`

Reply:
Agree, changed this, probably forgot this change during one of repases.